openslides/assignments/projector.py

```diff
@@ -27,3 +28,9 @@ def get_requirements(self, config_entry):
             for poll in assignment.polls.all().prefetch_related('options'):
                 for option in poll.options.all():
                     yield option.candidate
+                for speaker in assignment.agenda_item.speakers.filter(end_time=None):
```

Comment:
Do not yield speakers on this element. Yield them at the ListOfSpeakersOverlay projector element.